weave-js/src/components/Panel2/PanelPanel.tsx

```diff
@@ -54,7 +54,13 @@ import produce from 'immer';
 const inputType = {type: 'Panel' as const};
 type PanelPanelProps = Panel2.PanelProps<
   typeof inputType,
-  ChildPanelFullConfig
```

Comment:
`ChildPanelFullConfig` is not the correct type to pass in here. What actually comes through is `ChildPanelFullConfig['config']`, which is typed as `any`.

This incorrect typing wasn't an issue before because `PanelPanel`'s `config.config` has been `undefined` (until now) and we never tried to access any values on it.